README.md

```diff
@@ -57,6 +57,9 @@ To update Mist in the future, run:
     $ gulp update-nodes
 
 
+#### Options
+it is possible to pass the platform will be installed, see more [here](#platform)
```

Comment:
To stay consistent with the overall appearance you can also may want to change it to a sentence with punctuation.
Feel free to incorporate any of this as you see fit:
>It may be preferable to only download platform-specific nodes by passing the `--platform` flag, please refer to the [*options* section](#platform).